<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1356</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QGroupBox {
    border: 1px solid #CCCCCC;
    border-radius: 9px;
    margin-top: 8px;
}

QGroupBox::title {
    subcontrol-origin: margin;
    left: 10px;
    padding: 0 3px 0 3px;
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>6</number>
    </property>
    <property name="leftMargin">
     <number>6</number>
    </property>
    <property name="topMargin">
     <number>6</number>
    </property>
    <property name="rightMargin">
     <number>6</number>
    </property>
    <property name="bottomMargin">
     <number>6</number>
    </property>
    <item>
     <widget class="QGroupBox" name="mainGB">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>230</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QGroupBox {
    margin-top: 0px;
}</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QGroupBox" name="subdivGroupBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>210</y>
         <width>211</width>
         <height>251</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="leftMargin">
         <number>5</number>
        </property>
        <property name="topMargin">
         <number>5</number>
        </property>
        <property name="rightMargin">
         <number>5</number>
        </property>
        <property name="bottomMargin">
         <number>5</number>
        </property>
        <item>
         <widget class="QLabel" name="subdivTitleLabel">
          <property name="text">
           <string>Subdivision Settings</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="presetLabel">
          <property name="text">
           <string>Preset</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="netPresets">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <item>
           <property name="text">
            <string>Pentagon</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Basis</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>G</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="maskLabel">
          <property name="text">
           <string>Mask</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="subdivMask">
          <property name="text">
           <string>1 4 6 4 1</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="stepsLabel">
          <property name="text">
           <string>Subdivision Steps</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="subdivSteps"/>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="renderGroupBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>211</width>
         <height>191</height>
        </rect>
       </property>
       <widget class="QLabel" name="renderTitleLabel">
        <property name="geometry">
         <rect>
          <x>6</x>
          <y>6</y>
          <width>201</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Render Settings</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QCheckBox" name="controlNetCheckBox">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>92</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Control net</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="subdivCurveCheckBox">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>133</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>CPU Subdivision</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="curvatureCombCheckBox">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>100</y>
          <width>191</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Visualise curvature comb</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="tessellationCheckBox">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>130</y>
          <width>131</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>GPU Tessellation</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="highlight">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>160</y>
          <width>171</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Control point influence </string>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>480</y>
         <width>211</width>
         <height>91</height>
        </rect>
       </property>
       <property name="title">
        <string>Tessellation Settings</string>
       </property>
       <widget class="QLineEdit" name="lineEdit">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>191</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Choose tessellation level</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>91</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>No. of isolines</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="noIsoline">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>50</y>
          <width>42</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>91</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>No. of splits</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="noSplit">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>70</y>
          <width>42</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="MainView" name="mainView"/>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MainView</class>
   <extends>QOpenGLWidget</extends>
   <header>mainview.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>subdivCurveCheckBox</tabstop>
  <tabstop>netPresets</tabstop>
  <tabstop>subdivMask</tabstop>
  <tabstop>subdivSteps</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
